src/components/forms/Switch.js

```diff
@@ -6,27 +6,33 @@ const Switch = ({
   checked,
   onChange,
   srTitle,
+  name,
   ...rest
 }) => {
   return (
-    <HeadlessSwitch
-      checked={checked}
-      onChange={onChange}
-      className={classnames(
-        checked ? 'bg-primary-600' : 'bg-gray-200',
-        'relative inline-flex flex-shrink-0 h-6 w-11 border-2 border-transparent rounded-full cursor-pointer transition-colors ease-in-out duration-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary-500'
-      )}
-      {...rest}
-    >
-      <span className="sr-only">{srTitle}</span>
-      <span
-        aria-hidden="true"
+    <>
+      <HeadlessSwitch
+        checked={checked}
+        onChange={onChange}
         className={classnames(
-          checked ? 'translate-x-5' : 'translate-x-0',
-          'inline-block h-5 w-5 rounded-full bg-white shadow transform ring-0 transition ease-in-out duration-200'
+          checked ? 'bg-primary-600' : 'bg-gray-200',
+          'relative inline-flex flex-shrink-0 h-6 w-11 border-2 border-transparent rounded-full cursor-pointer transition-colors ease-in-out duration-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary-500'
```

Comment:
There are no real changes in these classes here. The diff is getting confused I think.